evaluator/autoware_control_evaluator/src/control_evaluator_node.cpp

```diff
@@ -260,6 +263,67 @@ void ControlEvaluatorNode::AddBoundaryDistanceMetricMsg(
   }
 }
 
+void ControlEvaluatorNode::AddUncrossableBoundaryDistanceMetricMsg(const Pose & ego_pose)
+{
+  namespace bdc_utils = autoware::boundary_departure_checker::utils;
+  constexpr auto extra_margin{5.0};
```

Comment:
A small comment to justify this margin would be nice.

Reply:
Added the comment, but not sure if it is descriptive enough https://github.com/autowarefoundation/autoware_universe/pull/10859/commits/4faeda846607c5a7a2d37a6af6f132f48545c375